control-plane/subcommand/connect-init/command_ent_test.go

```diff
@@ -9,111 +9,100 @@ import (
 	"os"
 	"testing"
 
-	"github.com/hashicorp/consul-k8s/control-plane/helper/test"
-	"github.com/hashicorp/consul-k8s/control-plane/namespaces"
-	"github.com/hashicorp/consul-k8s/control-plane/subcommand/common"
 	"github.com/hashicorp/consul/api"
 	"github.com/hashicorp/consul/sdk/testutil"
 	"github.com/mitchellh/cli"
 	"github.com/stretchr/testify/require"
+
+	connectinject "github.com/hashicorp/consul-k8s/control-plane/connect-inject"
+	"github.com/hashicorp/consul-k8s/control-plane/helper/test"
+	"github.com/hashicorp/consul-k8s/control-plane/namespaces"
+	"github.com/hashicorp/consul-k8s/control-plane/subcommand/common"
 )
 
-func TestRun_ServicePollingWithACLsAndTLSWithNamespaces(t *testing.T) {
+func TestRun_WithNamespaces(t *testing.T) {
```

Comment:
I've removed TLS test cases from this test and the command_test.go because I don't think there's any logic in the command that is tls-related  as it just takes whatever client  we configure from http flags. As a result, I don't think it'll be possible for these tests to fail if the code changes, and so I don't think there's any reason to keep them.